utils.py

```diff
@@ -158,6 +159,25 @@ def parse_from_and_to_dates(
     return from_time, until_time
 
 
+def resample_if_needed(s: pd.Series, sensor: Sensor, start: datetime, end: datetime) -> pd.Series:
```

Comment:
Looks to me like this function could go right to FlexMeasures? It would look good in a `utils/time_series` module  :)

Also, it deserves a simple test for both up- and downsampling, as it can be quite vital.

This code differs from code we already have in FM for upsampling and downsampling (see `api/utils/api_utis/upsample_values` or `api/common/utils/validators.py`) in that it accepts `pd.Series` and is not tied to data coming through the FM API.

Reply:
I'm not interested in moving this functionality into FlexMeasures. If I would have written this plugin from scratch, my strategy would be to get the data into a `BeliefsDataFrame` as fast as possible, so I'd then have access to the resampling functionality we already have in timely-beliefs.

Reply:
It seems to me you could have done that here as well. entsoe-py returns a `pd.Series`, which you could turn into a `BeliefsDataFrame`?